pkgs/desktops/xfce/default.nix

```diff
@@ -159,6 +159,7 @@ lib.makeScope pkgs.newScope (self: with self; {
 
   xfce4_power_manager_gtk3 = xfce4-power-manager.override { withGtk3 = true; };
 
+} // lib.optionalAttrs (config.skipAliases or false == false) {
```

Comment:
This is a little confusing to me... Would ```!(config.skipAliases or false)``` be okay?

Reply:
I'd better invert this option (```skipAliases``` -> ```allowAliases```) while no one uses it.
It has negation in the name and almost every its use is prefixed with ```!```

(```if config.skipAliases or false then {} else``` in "top-level/stage.nix" is also an exact equivalent of ```lib.optionalAttrs !(config.skipAliases or false)```)

Reply:
Yes that probably makes sense. It’s a little bt awkward because it’s not always obvious what the default value is.